src/scripts/createRuntimeChannels.ts

```diff
@@ -19,3 +19,14 @@ export function createRuntimeChannels() {
         writeFileSync(resolve(rootDir, `dist/provider/app.runtime-${channel}.json`), output, {encoding: 'utf8'});
     });
 }
+
+function getProviderManifestPath(rootDir: string): string {
```

Comment:
Is there ever a time when you'll pass in something thats not equal to `getRootDirectory()`?  Just wondering if we need this param at all